pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1194,23 +1194,30 @@ type PodAffinityChecker struct {
 	podLister      schedulerlisters.PodLister
 }
 
+// NewPodAffinityChecker returns a PodAffinityChecker.
+func NewPodAffinityChecker(sharedLister schedulerlisters.SharedLister) *PodAffinityChecker {
```

Comment:
for a next PR: PodAffinityChecker sounds like a really bad name. We can rename it once we move it to the plugin file.

Reply:
will not be needed at all.